tests/test_write_block.py

```diff
@@ -1,6 +1,10 @@
+# local
```

Comment:
I'm probably going to remove this line and the `3rd party` as they are not present in all the other files (and in general I don't know if the code base always follows the convention of this import order)